photon-server/src/main/java/org/photonvision/common/configuration/ConfigManager.java

```diff
@@ -433,4 +435,32 @@ private void checkSaveAndWrite() {
             saveToDisk();
         }
     }
+
+    public List<String> getSnapshots(String cameraUniqueName) {
```

Comment:
General thought - the String is playing double purpose of identifying the camera uniquely and being part of the file path where its snapshots live. Definitely functional. Architecturally, maybe pass in the whole camera object to the getSnapshot* functions? Or have the camera explicitly produce the `Path` objects where its snapshots should be?

Goal is just to separate out purposes - in case additional info beyond just the unique name is needed to create the path in the future.